Funcky/Internal/HandleException.cs

```diff
@@ -0,0 +1,19 @@
+namespace Funcky.Internal
+{
+    internal class HandleException<TException>
+            where TException : Exception
```

Comment:
Something went wrong with the indentation here
```suggestion
        where TException : Exception
```